components/shared/embedder/lib.rs

```diff
@@ -166,6 +164,13 @@ pub enum EmbedderMsg {
     ResizeTo(WebViewId, DeviceIntSize),
     /// Show dialog to user
     Prompt(WebViewId, PromptDefinition, PromptOrigin),
+    /// Request authentication for a load or navigation from the embedder.
+    RequestAuthentication(
+        WebViewId,
+        ServoUrl,
+        bool,
+        /* for proxy */ IpcSender<Option<AuthenticationResponse>>,
```

Comment:
The formatting here ended up weird. Maybe move the comment before the comma on the previous line?

Reply:
Okay. This is done now.